### DISCUSSION TITLE
How do I setup Discord Webhook?

### DISCUSSION THREAD

**Discussion by SuperFlyBry** [jef/streetmerchant#1170] [Help] [answered]
### Description

I have my discord script set up like this 

```
DISCORD_NOTIFY_GROUP="<@&7845040777113640>"
DISCORD_WEB_HOOK="https://www.discord.gg/x79TXGpq"
```

When I run the test it says

```
error:: couldn't send discord message Error sending webhook: 200 status code. Response: <!DOCTYPE html>
```

But if I use the discord URL on my web browser it works. I changed the ID and webhook in this thread for privacy.

**Comment by jef** [comment, marked as answer]
Take a look at Discord's [Intro to Webhooks](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks), that should get you going.

This is the main portion:

![image](https://user-images.githubusercontent.com/12074633/101225522-a4d2bf00-365f-11eb-8c35-d0f013e561d6.png)

![image](https://user-images.githubusercontent.com/12074633/101225550-b87e2580-365f-11eb-8be6-48b324b37916.png)

Use the URL there.

Good luck!

**Comment by SuperFlyBry** [comment]
Not sure if this is an old version of discord but there is no webhooks option for me

**Comment by SuperFlyBry** [comment]
Ah yes I found it under Integrations tab and it works now! Thank you!

**Comment by vincefrietema** [comment]
yo how do u do the notify group

**Comment by julius1504** [comment]
and for the role-ID (discord-notify-group) you need <@&ROLEID> (with the & sign), otherwise it wont notify the given roles

**Reply by vincefrietema** [reply]
ty

**Reply by vincefrietema** [reply]
it works now

**Comment by Nexetion** [comment]
How can I have it notify specific channels?

**Reply by jef** [reply]
https://github.com/jef/streetmerchant/discussions/1170#discussioncomment-169095

Create a webhook to that channel.
